nix/tools/generate_targets_rsa.py

```diff
@@ -0,0 +1,66 @@
+# generates a file to be used by the vegeta load testing tool
+import time
+import argparse
+import sys
+import random
+import jwt
+import jwcrypto.jwk as jwk
+
+URL = "http://postgrest"
+TOTAL_JWTS = 1000
```

Comment:
This should be 10K right? Going from your [previous comment](https://github.com/PostgREST/postgrest/pull/4118/files#r2125613116).
```suggestion
TOTAL_JWTS = 10000
```